cypress/integration/SidePanelLabs.spec.ts

```diff
@@ -0,0 +1,61 @@
+import * as h from '../helpers';
+
+describe('Side Panel', () => {
+  before(() => {
+    h.stories.visit();
+  });
+
+  ['Default', 'AsAside', 'AsDiv'].forEach(story => {
+    context(`given the "${story}" story is rendered`, () => {
+      beforeEach(() => {
+        h.stories.load('Testing|React/Labs/Side Panel/Cypress', story);
+      });
+
+      const name = /Accessible Label Name/i;
+
+      it('should not have any axe errors', () => {
+        cy.checkA11y();
+      });
+
+      it(`should have a button with an accessible name`, () => {
+        cy.findByRole('button', {name}).should('exist');
+      });
+
+      it(`should have a panel with a landmark role`, () => {
+        switch (story) {
+          case 'AsAside':
+            cy.findByRole('complementary', {name}).should('exist');
+            break;
+          default:
+            cy.findByRole('region', {name}).should('exist');
+        }
+      });
+
+      // TODO: How to test for aria-controls on the button
```

Comment:
Is this a need-to-have or a nice-to-have test?

Reply:
Good question, this is a nice-to-have a11y requirement at this point so I would probably factor it in as a "nice-to-have".

Reply:
Also, this is covered as a unit test when testing the hook.